config.nix

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I see these nix files are an update from the old `shell.nix` and `pkgs.nix` below, but I'm wondering why we have nix files in this repo at all? Do we do anything with nix here? If not, maybe we can take them out entirely?

Reply:
I was updating them to the current format, as they were already present in the repo, but I'm happy removing them as well. :+1;